src/test/resources/all/basic/multi_initialization.vpr

```diff
@@ -0,0 +1,17 @@
+// Any copyright is dedicated to the Public Domain.
+// http://creativecommons.org/publicdomain/zero/1.0/
+
+field n: Ref
+
+method get_refs() returns (x: Ref, y: Ref)
+    ensures acc(x.n)
+
+method foo() {
+  var a: Ref, b: Ref
+
+  var c: Ref, d: Ref := get_refs()
+  var e: Ref := new(n)
+
+  c.n, e := get_refs()
+  c.n.n := new(n)
```

Comment:
I think the syntax introduces some new corner cases that should have tests. E.g. I could have 
```
c.n.n, c := foo()
```
and this should assign to umm the ``n`` field of what ``c.n`` points to after (?) calling ``foo`` (which could change the value of ``c.n``) and before (?) changing the value of ``c``? Actually, regarding the latter point, what if it's 
```
c, c.n.n := foo() 
```
instead?
Currently, we reject
```
x, x := foo()
```
so maybe we need similar rules for field assignments? (Or maybe you added them and I didn't notice).

Reply:
Note to self: we just agreed in the Viper meeting that the semantics is that the assignment happens in parallel. Thus if were to do:
```
c.n c.n.n := foo()
```
One is assigning to `eval_pre_assign(c).n` and `eval_pre_assign(c.n).n` and so the behaviour is clear (we use the old `c.n`)